ssh_proxy_server/plugins/ssh/injectorshell.py

```diff
@@ -37,7 +38,7 @@ def __init__(self, session):
         self.injector_port = self.args.ssh_injector_port
         self.injector_sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self.injector_sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-        while True: # test/fix this
+        while True:     # TODO: test/fix this
             b = self.injector_sock.connect_ex((self.injector_ip, self.injector_port)) == 0
```

Comment:
Why are you try to connect to the port?
You are using port 0 as default. So you are using an random open port.

I would recommend to remove the option to specify a port and try to increase the port number, when the port is in use.